packages/trees/frontend/sirius-components-trees/src/treeitems/TreeItem.types.ts

```diff
@@ -24,6 +24,7 @@ export interface TreeItemProps {
   setSelection: (selection: Selection) => void;
   readOnly: boolean;
   textToHighlight: string | null;
-  isFilterEnabled: boolean;
+  textToFilter: string | null;
```

Comment:
What's the difference between `textToFilter` and `textToHighlight`?
From what I see,`textToFilter` is always equals to `textToHighlight`, but is only valued when the filter of the FilterBar is enabled right?
So in this case why didn't let the `isFilterEnabled: boolean;`?

Reply:
In order to keep the tree as dump as possible, I thought it wasn't the responsibility of this component to decide that the highlighted text is always the same as the filtered text. What do you think about it?

Reply:
I don't see a case where this could happen, but ok.